array/move_zeros_to_end.py

```diff
@@ -0,0 +1,24 @@
+"""
+Write an algorithm that takes an array and moves all of the zeros to the end, preserving the order of the other elements.
+move_zeros([false,1,0,1,2,0,1,3,"a"]) # returns[false,1,1,2,1,3,"a",0,0]
+
+The time complexity of the below algorithm is O(n).
+"""
+
+def move_zeros(array):
+    if len(array) < 1:
+        return array
+    else:
+        list = []
+        zeros = 0
+
+        for i in array:
+            if not i and type(i) is int or type(i) is float:
```

Comment:
replace with `if i == 0:`

Reply:
That won't work If we have False as one of the element in our list, for instance, 
`list[0,"a", False]` should be == to `list["a",False, 0]` but If you make `i == 0` then False will be added to the end which is not what we want.

Reply:
yeah @SaadBenn , you are right. But the current version seems a bit akward.
I guess we should use 
```python
if i is 0 or i is 0.0:
``` 
instead.